src/presence/useConnectionStatus.ts

```diff
@@ -25,23 +25,23 @@ export const useConnectionStatus = (
 
   useEffect(() => {
     if (ditto) {
-      peersObserverRef.current = ditto.observePeers((peers) => {
-        const nextActiveConnections = peers.reduce(
-          (acc: Set<PresenceConnectionType>, peer: RemotePeer) => {
-            peer.connections.forEach((connectionType) =>
-              acc.add(connectionType),
-            )
+      peersObserverRef.current = ditto.presence.observe(
+        ({ remotePeers }: { remotePeers: Peer[] }) => {
```

Comment:
Do we not provide types for the callback function in the `presence.observe` call at the SDK level? Mostly curious why the explicit typing was required

Reply:
You're right, the explicit types are not needed. I've pushed a commit removing them. Thanks for catching this!